conf.py

```diff
@@ -96,8 +96,8 @@
 html_favicon = './_static/images/favicon.ico'
 html_theme = 'furo'
 html_theme_options = {
-    "light_logo": "images/logoLightPride.png",
-    "dark_logo": "images/logoDarkPride.png",
+    "light_logo": "images/logoLight.png",
+    "dark_logo": "images/logoDark.png",
```

Comment:
Not sure if this is meant - when do we undo the pride crab?

Reply:
Was an accident commit :) Is now all good since we reverted back from Pride logo.